src/main/java/io/strimzi/kafka/bridge/KafkaBridgeAdmin.java

```diff
@@ -84,6 +86,31 @@ public CompletionStage<Set<String>> listTopics() {
         return promise;
     }
 
+    /**
+     * Creates a topic with given name
+     *
+     * @param topicName topic name to create
+     * @param partitions number of partitions
+     * @param replicationFactor replication factor
+     * @return a CompletionStage Void
+     */
+    public CompletionStage<Void> createTopic(String topicName, int partitions, short replicationFactor) {
```

Comment:
why are you enforcing the user to provide partitions and replicationFactor? The `NewTopic` class provides also a way to have them as optional, I think the user should be able to ask for creating a topic by using the defaults as well.